FrontEnd/src/components/basics/Amenities/Amenities.tsx

```diff
@@ -102,8 +103,8 @@ const Amenities: FunctionComponent<AmenitiesProps> = ({
   return (
     <div className={cn(styles.wrapperDefault, className)}>
       {selected.map((s) => {
-        const SelectedIcon = amenityToIcon[s];
-        const Icon = () => (variant !== 'onlyLabel' ? <SelectedIcon /> : null);
+        const SelectedIcon = <FallbackIcon amenity={s} />;
```

Comment:
nit: change to `selectedIcon` (capitalize when it's used as a Component but this isn't)